Project.toml

```diff
@@ -10,7 +10,8 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 julia = "1.6,1.7"
 
 [extras]
+Suppressor = "fd094767-a336-5f1f-9728-57cf17d0bbfb"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [targets]
-test = ["Test"]
+test = ["Test", "Suppressor"]
```

Comment:
please let me know if this is the incorrect way of specifying test dependencies!

Reply:
Actually yes, test dependencies are supposed to be somewhere else: https://github.com/nathanaelbosch/ProbNumDiffEq.jl/blob/main/test/Project.toml

TL;DR:
```
]activate test
add Suppressor
```

Reply:
Thanks! I updated it.